rtl_433/src/util.c:117:4: error: non-void function 'local_time_str' should return a value [-Wreturn-type]

I get the following error during the compilation of this program using `Apple LLVM version 8.0.0 (clang-800.0.42.1)`.

```
[  1%] Building C object src/CMakeFiles/rtl_433.dir/util.c.o
/Users/casa/rtl_433/src/util.c:117:4: error: non-void function 'local_time_str' should return a value [-Wreturn-type]
                        return;
                        ^
1 error generated.
```

--- a/src/util.c
+++ b/src/util.c
@@ -114,7 +114,7 @@ char* local_time_str(time_t time_secs, char *buf) {
 		extern float sample_file_pos;
 		if (sample_file_pos != -1.0) {
 			snprintf(buf, LOCAL_TIME_BUFLEN, "@%fs", sample_file_pos);
-			return 0;
+			return buf;
 		}
 		time(&etime);
 	} else {

--- a/src/util.c
+++ b/src/util.c
@@ -114,7 +114,7 @@ char* local_time_str(time_t time_secs, char *buf) {
 		extern float sample_file_pos;
 		if (sample_file_pos != -1.0) {
 			snprintf(buf, LOCAL_TIME_BUFLEN, "@%fs", sample_file_pos);
-			return;
+			return 0;
 		}
 		time(&etime);
 	} else {

